src/ui/components/ColorPreview.svelte

```diff
@@ -0,0 +1,20 @@
+<script lang="ts">
+  import BasicColorPreviewIcon from '~ui/components/BasicColorPreviewIcon.svelte';
+  import LayeredColorPreviewIcon from '~ui/components/LayeredColorPreviewIcon.svelte';
+
+  interface Props {
```

Comment:
Why you define extra `interface`?

For instance it can be done by

```ts
const { borderColor, indicatorColor, isBlended }: {
  borderColor: string;
  indicatorColor: string;
  isBlended: boolean;
} = $props();
```